app/src/main/kotlin/at/bitfire/davdroid/ui/setup/AdvancedLogin.kt

```diff
@@ -88,8 +89,7 @@ fun AdvancedLoginScreen(
     onSetUrl: (String) -> Unit = {},
     username: String,
     onSetUsername: (String) -> Unit = {},
-    password: String,
-    onSetPassword: (String) -> Unit = {},
+    password: TextFieldState,
```

Comment:
What's the reason to pass the `password` as a state to the `PasswordTextField`? It confuses me that the username uses another method of being passed around (get/set callbacks) as the password (pass mutable state).